src/improvements/tasks/eth/005-opcm-upgrade-v300-uni/config.toml

```diff
@@ -0,0 +1,42 @@
+l2chains = [
+    {name = "Unichain Mainnet", chainId = 130}
+]
+
+templateName = "OPCMUpgradeV300"
+
+[[opcmUpgrades]]
+chainId = 130
+# https://oplabs.notion.site/Upgrade-14-MTCannon-1d6f153ee1628024af26cd0098d3bdfe
+absolutePrestate = "0x03ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee405"
+# PROXYA-10: Proxy admin owner must be l1PAOMultisig - This is OK because it is checking for the OP Sepolia PAO.
+# SYSCON-20: System config gas limit must be 60,000,000 - This check is wrong. The config requirement is no greater than 200,000,000.
+#  https://specs.optimism.io/protocol/configurability.html?highlight=configuration#gas-limit
+# DF-30: Dispute factory owner must be l1PAOMultisig - It is checking for the OP Sepolia PAO.
+# PDDG-DWETH-30: Delayed WETH owner must be l1PAOMultisig (for permissioned dispute game) - It is checking for the OP Sepolia PAO
+# PDDG-ANCHORP-40: The anchor state registry's permissioned root is not 0xdead000000000000000000000000000000000000000000000000000000000000
+# PDDG-120: Permissioned dispute game challenger must match challenger address - It is checking for the OP Sepolia Challenger
+# PLDG-DWETH-30: Delayed WETH owner must be l1PAOMultisig (for permissioned dispute game) - It is checking for the OP Sepolia PAO
+# PLDG-ANCHORP-40: The anchor state registry's permissioned root is not 0xdead000000000000000000000000000000000000000000000000000000000000
+expectedValidationErrors = "PROXYA-10,SYSCON-20,DF-30,PDDG-DWETH-30,PDDG-DWETH-40,PDDG-ANCHORP-40,PLDG-DWETH-30,PLDG-DWETH-40,PLDG-ANCHORP-40"
+
+
+[addresses]
+OPCM = "0x3a1f523a4bc09cd344a2745a108bb0398288094f" # v1.9.0 https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml#L22
+StandardValidatorV300 = "0x1b8f0e76bb49b60dffbe3b847e25d6429ebff3e6" # V300 not deployed/standardized for mainnet, using V200 from previous task
+ChildSafe1 = "0xb0c4C487C5cf6d67807Bc2008c66fa7e2cE744EC"
+ChildSafe2 = "0x847B5c174615B1B7fDF770882256e2D3E95b9D92"
+ChildSafe3 = "0xc2819DC788505Aac350142A7A707BF9D03E3Bd03"
+
+[stateOverrides]
+0x6d5B183F538ABB8572F5cD17109c617b994D5833 = [ # L1PAO: 3
```

Comment:
I noticed this wasn't added to the notion doc. Good that we have it here but we should push whoever owns the nonce co-ordination to manage these better.